src/data/helper/dimensionHelper.ts

```diff
@@ -169,6 +173,10 @@ export function summarizeDimensions(
         defaultedLabel = encodeLabel.slice();
     }
 
+    if (!defaultedLabel || !defaultedLabel.length) {
```

Comment:
The `!defaultedLabel` will always be `false`. 
Only `if (!defaultedLabel.length) {...}` is enough